README.md

```diff
@@ -35,6 +35,7 @@ Method    | Path
 **GET**   | `/company/{companyNumber}/registered-email-address`                                  | Calls service to retrieve the registered email address for a company
 **GET**   | `/overseas-entity/{companyNumber}/managing-officers`                                 | Calls service to retrieve the managing officers for an overseas entity
 **GET**   | `/overseas-entity/{companyNumber}/beneficial-owners`                                 | Calls service to retrieve the beneficial owner data for an overseas entity
+**GET**   |`/overseas-entity/{oeNumber}/trusts/links`                                            |Calls service to retrieve the link data for a trust
```

Comment:
Please add the relevant spacing, also we could update the description to include 'overseas entity'
```suggestion
**GET**   | `/overseas-entity/{oeNumber}/trusts/links`                                            | Calls service to retrieve the trust link data for an overseas entity
```